app/components/schools/ects/listing_card_component.rb

```diff
@@ -19,13 +19,27 @@ def appropriate_body_row
       def delivery_partner_row
         return if ect.school_led_training_programme?
 
-        { key: { text: 'Delivery partner' }, value: { text: latest_delivery_partner_name(ect) } }
+        {
+          key: { text: 'Delivery partner' },
+          value: {
+            text: latest_training_period_only_expression_of_interest? ? 'Their lead provider will confirm this' : latest_delivery_partner_name(ect)
+          }
+        }
       end
 
       def lead_provider_row
         return if ect.school_led_training_programme?
 
-        { key: { text: 'Lead provider' }, value: { text: latest_lead_provider_name(ect) } }
+        {
+          key: { text: 'Lead provider' },
+          value: {
+            text: latest_training_period_only_expression_of_interest? ? latest_eoi_lead_provider_name(ect) : latest_lead_provider_name(ect)
```

Comment:
Same here, it's long for a ternary statement.

Reply:
Done